macros/sfdc_formula_view_sql.sql

```diff
@@ -1,7 +1,8 @@
-{%- macro sfdc_formula_view_sql(join_to_table, source_name = 'salesforce') -%}
+{%- macro sfdc_formula_view_sql(join_to_table, source_name = 'salesforce', inclusion_fields=none) -%}
 
+    {%- set not_null_value = true -%}
```

Comment:
why should this be harcoded as true here?

Reply:
This was a result of the functionality of dbt was confusing me. The argument was not being passed along through the macros and I decided to hard code it. I want to check back in with this to see if I was doing something incorrectly that would fix this and result in not needing the hard code.